BuilderES5.js

```diff
@@ -0,0 +1,15 @@
+export default function BuilderES5 (initialValue) {
+  this._currentResult = initialValue
+}
+
+BuilderES5.prototype.plus = function (...values) {
```

Comment:
ES5 doesn't have rest parameters, so it's better to use here something from ES5

Reply:
I've used `arguments` here instead. https://github.com/khveska/inheritance/blob/0d81f3c268fe87aaa8fab9a910152130b678ad09/BuilderES5.js#L5-L10 Not sure if my approach is a legit way to replace rest parameters though.